SubtitlesAPI/ILocalization.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System.Collections.Generic;
+
+namespace SubtitlesAPI;
+
+/// <summary>
+/// An interface to easily implement new localizations
+/// </summary>
+public interface ILocalization
```

Comment:
Considering changing the interface name to avoid confusion with broader localization. Maybe something like ISubtitleLocalization would work?

Reply:
Good callout! Renamed.